web-server-doc/web-server/scribblings/formlets.scrbl

```diff
@@ -505,6 +511,11 @@ a list of elements of the sequence.
 
 @defthing[input-int (formlet/c number?)]{
   Equivalent to @racket[(to-number input-string)].
+  Note that, despite the name, the result
+  @bold{is not guaranteed to be an integer.}
+  
+  @history[#:changed "1.3"
+           "Weakened result contract to allow any number."]
```

Comment:
I do not understand the rationale behind <https://github.com/racket/web-server/commit/4cfb68e> (I couldn't find a PR 15447), but that is what it did.